test/manual/testplan.md

```diff
@@ -0,0 +1,90 @@
+# OpenCloud Desktop Client Manual Test Plan
+
+## Account Management
+
+Try everything in `Setup and Configuration` once with Keycloak and once with Lico (the single binary builtin IdP).
+
+### Setup and Configuration
+* Add a new account with valid credentials
+* Attempt to add an account with invalid credentials
+* Skip sync folder configuration during setup
+* Configure multiple sync connections for one account
+
+### Connection Management
+* Pause and resume synchronization
+* Remove a folder sync connection
+* Remove an account
+
+## Basic Synchronization
+### Basic File Operations
+* Create a new file locally and verify it syncs to server
+* Download files from the server to local machine
+* Modify a file locally and verify changes sync to server
+* Modify a file on server and verify changes sync to local machine
+* Delete a file locally and verify deletion syncs to server
+* Delete a file on server and verify deletion syncs to local machine
+
+### Folder Operations
+* Create empty folders and verify they sync
+* Create nested folder structures and verify they sync
+* Sync folders with blanks or special characters in the name
+* Verify long folder names sync correctly
+* Delete folders and verify deletion syncs
+
+### File Conflict Management
+* Create a file conflict by modifying the same file both locally and on server
+* Verify conflict resolution creates a proper conflict file
+* Test conflict resolution options
+
+## Advanced Synchronization
+### Selective Sync
+* Choose specific folders to sync during initial setup
+* Change selective sync settings for existing account
+* Verify unselected folders don't sync
+* Test sorting folder list by name and size
+
+### Large Files and Folders
+* Sync files of different sizes (1MB, 100MB, 1GB)
+* Sync folders containing many files (500+ files)
+* Sync deeply nested folder structures
+
+### Special Files
+* Sync files with various extensions (.txt, .jpg, .png, .pdf, .mp3, .mp4)
+* Test with very long filenames
+* Test with special characters in filenames
+* Test with files having spaces in names
+
+## Sharing
+### Public Link Sharing
```

Comment:
Sharing is web only, so we could only verify that the received share is synced properly?

Reply:
Removed that bogus and rephrased it to received shares.